script.js

```diff
@@ -19,48 +19,212 @@ for (i = 0; i < navLink.length; i++){
+        <a href=${data[0].linkversion_link} class="small-screen-link">See Live <i class="fa-solid fa-paper-plane fa"></i></a>
+        <a href=${data[0].sourceCode_link} class="small-screen-link">See Source <i class="fa-brands fa-github fa"></i></a>  
+    </div>
 </div>`;
-const closeCard = document.querySelector('.close_popup');
+const closeCard = document.querySelector('.close-popup');
 closeCard.addEventListener('click',()=>{
-  showCardDiv.innerHTML="";
+    popupCover.classList.remove('open');
+    cardHolder.innerHTML="";
     console.log('close  card');
 }) 
 };
+
+
+
 const showProjectBtns = Array.from(document.querySelectorAll('.seeProjectsBtn'));
-for(const showProjectBtn of showProjectBtns){
-    showProjectBtn.addEventListener('click',createCardUI)
-};
\ No newline at end of file
+for(let i=0;i<showProjectBtns.length;i++){
+    showProjectBtns[i].addEventListener('click',()=>{
+        createCardUI(i);
+    })
+}
+
+// day3 form validation
+//------------------------------------------
+const form  = document.querySelector('.form');
+const submitBtn = document.querySelector('.submit_btn');
+const errMessage = document.querySelector('.err-msg');
+const resetBtn  = document.querySelector('.reset_btn');
+// taking values
+const firstName = document.querySelector('.first-name');
+const lastName= document.querySelector('.last-name');
+const comments = document.querySelector('.message');
+const email= document.querySelector('.email-address');
+function takeValuesfromInput(){
+    const firstNameValue = firstName.value;
+    const lastNameValue = lastName.value;
+    const commentsValue = comments.value;
+    const emailValue = email.value;
+    return { firstNameValue,lastNameValue,commentsValue,emailValue };
+}
+// SHOW ERRO FUNCTION
+function showEror(message,displayStyle){
+    const errHolder = document.createElement('p');
+    errHolder.classList="err-msg";
+    errHolder.innerHTML=message;
+    errHolder.style.display=displayStyle;
+    form.appendChild(errHolder);
+    setTimeout(() => {
+        errHolder.style.display='none';
+    }, 3000);
+}
+function clearForm(){
+    firstName.value='';
+    lastName.value='';
+    comments.value='';
+    email.value='';
+}
+form.addEventListener('submit',(e)=>{
+    e.preventDefault();
+   console.log('sumbit');
+    // take values
+    const {firstNameValue,lastNameValue,commentsValue,emailValue} = takeValuesfromInput(); 
+    saveInLocalStorage('data',{firstNameValue,lastNameValue,commentsValue,emailValue});
+    const urEmail = emailValue.toLowerCase();
+    console.log(urEmail,emailValue);
+    if(emailValue!==urEmail){
+        showEror("Email should be in lower case","block");
+    }
+    else if( firstNameValue==""||lastName==""||commentsValue==""||emailValue=="" ){
+        showEror("Inputs shouldn't be empty!","block");
+        console.log('eror');
+    }
+    else{
+        console.log('err');
+    }
+    // clear input
+    clearForm();
+})
```

Comment:
Kindly make sure that the form submission is working properly. Currently both `Get in touch` and `Reset` buttons are clearing the form inputs.